.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,24 @@
+name: BrowserStack Acceptance Tests
+
+on: [push, pull_request]
+
+jobs:
+  run-acceptance-tests:
+    name: BrowserStack Acceptance Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: BrowserStack Environment Setup
+        uses: 'browserstack/github-actions/setup-env@master'
+        with:
+          username:  ${{ secrets.BROWSERSTACK_USERNAME }}
+          access-key: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+          project-name: 'Answers Hitchhiker Theme'
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm ci
```

Comment:
I think we should group together tests under the same PR at https://automate.browserstack.com/dashboard/v2/ 
Currently we're getting one build per browser tested, with one session each, as opposed to
having multiple sessions within a single build
the SDK does this by exporting some browserstack env vars
https://github.com/yext/answers-search-ui/blob/fc4fd42f94d2a693f727b2b480611b8ba790affb/.circleci/run_browserstack_acceptance.sh

We could also try using a browserstack-config.json which I think would be slightly cleaner. I can't remember why I didn't do this originally for the SDK.
https://github.com/DevExpress/testcafe-browser-provider-browserstack#configuration-file
https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs/testcafe#executing-tests-on-browserstack